cmd/run.go

```diff
@@ -82,12 +82,28 @@ func SetupRunFlags(cmd *cobra.Command) {
 		- hypersync: Will sync by downloading historical state, and will NOT
 		  download historical blocks. Can only be set if HyperSync is true.`)
 
+	// PoS Validator
+	cmd.PersistentFlags().String("pos-validator-seed", "", "A BIP39 seed phrase used to generate the private key of the "+
+		"Proof of Stake validator. Setting this flag automatically makes the node run as a Proof of Stake Validator.")
+	cmd.PersistentFlags().Uint64("pos-block-production-interval-milliseconds", 1500, "The interval in milliseconds "+
```

Comment:
Hmm... is it better for these to be flags or for them to be ParamUpdater? I'm thinking about how we would change these values later, for example if we want to reduce the block production interval to 1s or even faster if we think the network can handle it.

Right now, there's not really a good upgrade path here. You would need everyone to upgrade their flags at the exact same time or else the network would break. I think originally we said we'd put these in ParamUpdater for this reason, since it can safely change these values.

Reply:
Separate question is whether we should start this number low and increase it OR start it high and decrease it. I thought about it, and it feels like starting LOW is actually better. If we start it low, we'll encounter issues in devnet and know what our "limits" are. However, if we start it high we may be running at too generous of a buffer and always be too afraid to reduce it.

This is also something that seems easier to test with ParamUpdater, since you can play with values quickly while the network is live and see what happens, rather than having to reboot the whole network.